owasp-top10-2017-apps/a1/mongection/src/app.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const mongoose = require('mongoose');
+const db = require('./db');
+
+const server = express();
+
+const PORT = 9995;
```

Comment:
Hey, @vitor-fernandes, thanks so much for the contribution!

Would you mind changing the app's port from `9995` to `10001`? 
We are trying to make all the ports more intuitive for developers by having them use the same one as the app's category.

Reply:
I'll change now :)